library/eclipse-temurin

```diff
@@ -134,261 +134,224 @@ Constraints: nanoserver-1809, windowsservercore-1809
+Tags: 11.0.21_9-jre-windowsservercore-ltsc2022, 11-jre-windowsservercore-ltsc2022
+SharedTags: 11.0.21_9-jre-windowsservercore, 11-jre-windowsservercore, 11.0.21_9-jre, 11-jre
 Architectures: windows-amd64
-GitCommit: 25f458a0991f58bb7ed0ec9817e026a40d559c38
+GitCommit: 305dacd0cb9d4d45b2234a32928e5708e32f4362
 Directory: 11/jre/windows/windowsservercore-ltsc2022
-File: Dockerfile.releases.full
 Constraints: windowsservercore-ltsc2022
 
-Tags: 11.0.20.1_1-jre-nanoserver-ltsc2022, 11-jre-nanoserver-ltsc2022
-SharedTags: 11.0.20.1_1-jre-nanoserver, 11-jre-nanoserver
+Tags: 11.0.21_9-jre-nanoserver-ltsc2022, 11-jre-nanoserver-ltsc2022
+SharedTags: 11.0.21_9-jre-nanoserver, 11-jre-nanoserver
 Architectures: windows-amd64
-GitCommit: 25f458a0991f58bb7ed0ec9817e026a40d559c38
+GitCommit: 305dacd0cb9d4d45b2234a32928e5708e32f4362
 Directory: 11/jre/windows/nanoserver-ltsc2022
-File: Dockerfile.releases.full
 Constraints: nanoserver-ltsc2022, windowsservercore-ltsc2022
 
-Tags: 11.0.20.1_1-jre-windowsservercore-1809, 11-jre-windowsservercore-1809
-SharedTags: 11.0.20.1_1-jre-windowsservercore, 11-jre-windowsservercore, 11.0.20.1_1-jre, 11-jre
+Tags: 11.0.21_9-jre-windowsservercore-1809, 11-jre-windowsservercore-1809
+SharedTags: 11.0.21_9-jre-windowsservercore, 11-jre-windowsservercore, 11.0.21_9-jre, 11-jre
 Architectures: windows-amd64
-GitCommit: 25f458a0991f58bb7ed0ec9817e026a40d559c38
+GitCommit: 305dacd0cb9d4d45b2234a32928e5708e32f4362
 Directory: 11/jre/windows/windowsservercore-1809
-File: Dockerfile.releases.full
 Constraints: windowsservercore-1809
 
-Tags: 11.0.20.1_1-jre-nanoserver-1809, 11-jre-nanoserver-1809
-SharedTags: 11.0.20.1_1-jre-nanoserver, 11-jre-nanoserver
+Tags: 11.0.21_9-jre-nanoserver-1809, 11-jre-nanoserver-1809
+SharedTags: 11.0.21_9-jre-nanoserver, 11-jre-nanoserver
 Architectures: windows-amd64
-GitCommit: 25f458a0991f58bb7ed0ec9817e026a40d559c38
+GitCommit: 305dacd0cb9d4d45b2234a32928e5708e32f4362
 Directory: 11/jre/windows/nanoserver-1809
-File: Dockerfile.releases.full
 Constraints: nanoserver-1809, windowsservercore-1809
 
 
 #------------------------------v17 images---------------------------------
-Tags: 17.0.8.1_1-jdk-alpine, 17-jdk-alpine, 17-alpine
+Tags: 17.0.9_9-jdk-alpine, 17-jdk-alpine, 17-alpine
 Architectures: amd64
-GitCommit: 25f458a0991f58bb7ed0ec9817e026a40d559c38
+GitCommit: 305dacd0cb9d4d45b2234a32928e5708e32f4362
 Directory: 17/jdk/alpine
-File: Dockerfile.releases.full
 
-Tags: 17.0.8.1_1-jdk-focal, 17-jdk-focal, 17-focal
+Tags: 17.0.9_9-jdk-focal, 17-jdk-focal, 17-focal
 Architectures: amd64, arm32v7, arm64v8, ppc64le, s390x
-GitCommit: 25f458a0991f58bb7ed0ec9817e026a40d559c38
+GitCommit: 305dacd0cb9d4d45b2234a32928e5708e32f4362
 Directory: 17/jdk/ubuntu/focal
-File: Dockerfile.releases.full
 
-Tags: 17.0.8.1_1-jdk-jammy, 17-jdk-jammy, 17-jammy
-SharedTags: 17.0.8.1_1-jdk, 17-jdk, 17
+Tags: 17.0.9_9-jdk-jammy, 17-jdk-jammy, 17-jammy
+SharedTags: 17.0.9_9-jdk, 17-jdk, 17
```

Comment:
The Windows images don't exactly line up with the `17.0.9_9-jdk` and `-jre` tags, since they have a slightly different version number:
```yaml
SharedTags: 17.0.9_9.1-jdk-windowsservercore, 17-jdk-windowsservercore, 17-windowsservercore, 17.0.9_9.1-jdk, 17-jdk, 17

SharedTags: 17.0.9_9.1-jre-windowsservercore, 17-jre-windowsservercore, 17.0.9_9.1-jre, 17-jre
```

What this means is that `17.0.9_9-jdk` and `17.0.9_9-jre` won't have any Windows images in the image index (i.e., can't `docker pull 17.0.9_9-jdk` on Windows) and `17.0.9_9.1-jdk` and `17.0.9_9.1-jre` will be Windows-only tags. Is that acceptable?

Reply:
good spot, I'll fix this now

Reply:
@yosifkit updated PTAL